packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -467,6 +471,13 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
     });
   }
 
+  /* @conditional-compile-remove(unsupported-browser) */
+  public async getEnvironmentInfo(): Promise<EnvironmentInfo> {
+    return await this.asyncTeeErrorToEventEmitter(async () => {
+      return this.callClient.feature(Features.DebugInfo).getEnvironmentInfo();
```

Comment:
this is not being awaited, either:
```suggestion
      return this.callClient.feature(Features.DebugInfo).getEnvironmentInfo;
```
or
```suggestion
      return await this.callClient.feature(Features.DebugInfo).getEnvironmentInfo();
```